### PR TITLE
feat-US34063: Added Configuration Contract for fetching the transactions fee available while creating the pool

### PR DESCRIPTION
**Pull request by roshan-bisht-lab49**

### CODE REVIEW

File: src/dex-ui/store/poolsSlice/poolsSlice.ts [hunk 1]
```diff
@@ -211,6 +217,41 @@ const createPoolsSlice: PoolsSlice = (set, get): PoolsStore => {
       }
       app.setFeaturesAsLoaded(["addLiquidityTransactionState"]);
     },
+    fetchCreatePoolTransactionFee: async () => {
```
Comment (damiano-melcarne-lab49) [inline src/dex-ui/store/poolsSlice/poolsSlice.ts:220]: Please move this to a React Query hook. We need to move away from using Zustand for managing server state.

**Review by damiano-melcarne-lab49 — CHANGES_REQUESTED** [review]

File: src/dex-ui/store/poolsSlice/poolsSlice.ts (hunk 1 above)
Comment (roshan-bisht-lab49) [reply src/dex-ui/store/poolsSlice/poolsSlice.ts]: Done

File: src/dex-ui-components/base/Dropdown/Dropdown.tsx [hunk 2]
```diff
@@ -18,17 +19,19 @@ const DropdownSelector = (props: DropdownSelectProps) => {
...
-              <option key={input.value} value={input.value}>
-                {input.label}
-              </option>
-            );
-          })
-        : undefined}
-    </Select>,
+    <Skeleton speed={0.4} fadeDuration={0} isLoaded={!props.isLoading}>
+      <Select variant="dropDownSelector" value={value} icon={<ArrowDropDownIcon />} {...props.selectControls}>
+        {data
```
Comment (damiano-melcarne-lab49) [inline src/dex-ui-components/base/Dropdown/Dropdown.tsx:24]: You can use the optional chaining for shorthand here: `data?.map`

**Review by damiano-melcarne-lab49 — APPROVED** [review]

File: src/dex-ui-components/base/Dropdown/Dropdown.tsx (hunk 2 above)
Comment (roshan-bisht-lab49) [reply src/dex-ui-components/base/Dropdown/Dropdown.tsx]: Done

**Pull request merged**